### PR TITLE
Fix for issue #1595

### PR DESCRIPTION
**Pull request by mbudiu-vmw**
The fix is a one-liner (line 238 in simplifyDefUse), but I renamed a bunch of functions because TAGS were not useful to navigate the code at all.

### CODE REVIEW

**Comment by mbudiu-vmw** [comment]
Anyone want to review this one-line fix (plus some code beautification)?

File: frontends/p4/def_use.cpp [hunk 1]
```diff
@@ -817,8 +817,8 @@ bool ComputeWriteSet::preorder(const IR::AssignmentStatement* statement) {
 bool ComputeWriteSet::preorder(const IR::SwitchStatement* statement) {
     LOG3("CWS Visiting " << dbp(statement));
     visit(statement->expression);
-    auto locs = get(statement->expression);
-    auto defs = currentDefinitions->writes(getProgramPoint(), locs);
+    auto locs = getWrites(statement->expression);
+    auto defs = currentDefinitions->writes(getProgramPoint(statement->expression), locs);
```
Comment (ChrisDodd) [inline frontends/p4/def_use.cpp]: What about this change?  It is not (just) a renaming -- there's a real change here.  Is it intentional?

**Comment by ChrisDodd** [comment]
This would be much easier to review if you have separate commits for the rename change vs the bugfix change (when viewing the 'files changed' you can select single commits to review independently)

**Comment by mbudiu-vmw** [comment]
Yes, the renaming was part of the debugging process. Next time I will commit the fix separately. You should trust that the c++ typechecker checks that the renaming is right.

**Pull request merged**